documentation/service-rules/school-admins.md

```diff
@@ -117,11 +133,243 @@ We ask this question so we can get a better understanding of when the ECT is sta
+
+If we can't find the mentor's record in the Teaching Record System API, but the TRN does exist, we ask for that mentor's national insurance number instead. We do this because potentially the date of birth stored on the record in the TRS is incorrect, or the one the school holds might be. It makes it more likely for the user to find mentor's ECT's teacher record and confirm they know them.
+
+If we still cannot confirm the user knows both the TRN and either national insurance number or date of birth, we tell them the record cannot be found.
+
+Neither date of birth or national insurance number should be stored longer-term. It is just used for the initial finding and checking of a mentor's record in the TRS. You can read more [about our reasoning for this here](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/no-longer-storing-date-of-birth/).
+
+If the mentor does not have teacher reference number, we take school users to a page that informs they of how to either [Find a lost TRN](https://find-a-lost-trn.education.gov.uk/start) or [request a new TRN for the mentor](https://www.gov.uk/guidance/teacher-reference-number-trn#if-youre-a-mentor-for-a-trainee-or-early-career-teacher).
+
+### Confirming or correcting a mentor's name
+
+Once we've decided the mentor can be added to the service, we play back the name from the TRA's Teaching Record System to the user. This is so we can show them it's linked to their teacher record and that they must be intending to register that person.
+
+We give users the option to either:
+
+* confirm the name is correct and continue
+* correct the name and continue
+
+We did this because:
+
+* we know the name held in the TRS is often out of date
+* we want lead providers to have correct contact details for mentors
+* we don't want to block schools from registering someone because the name of the mentor may have been updated
+
+If the name is corrected, we will continue to show the corrected name in the service from this point. We will continue to store the name from the TRS, so we can monitor if this feature is being used correctly, and it's not being overwritten with names that are completely different.
+
+You can read more about why we chose to change how a mentor's name is gathered [in the design history entry here](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/correcting-names/).
+
+### Registering a mentor who has been registered before at another school
+
+We still need to define what happens here.
+
+### Giving a mentor's email address
+
+The school user is asked for that mentor's email address.
+
+We check the email address given and make sure it doesn't exist for an ongoing ECT or mentor record with a different TRN. This would mean any emails that are attached to an open ect_at_school period. This is because we shouldn't have email addresses that are the same for entirely different people, when both are are still undergoing training.
```

Comment:
```suggestion
We check the email address given and make sure it doesn't exist for an ongoing ECT or mentor record with a different TRN. This would mean any emails that are attached to an open `ect_at_school_period`. This is because we shouldn't have email addresses that are the same for entirely different people, when both are are still undergoing training.
```

Reply:
Fixed in: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/325/commits/5bb7b5b433f25f782d9748df41b67d17cf49147d